va/va.h

```diff
@@ -1273,6 +1328,36 @@ typedef union _VAConfigAttribValContextPriority{
     uint32_t value;
 }VAConfigAttribValContextPriority;
 
+/** @name Attribute values for VAConfigAttribProtectedContentCipherAlgorithm */
+/** \brief AES cipher */
+#define VA_PC_CIPHER_AES                    0x00000001
+
+/** @name Attribute values for VAConfigAttribProtectedContentCipherBlockSize */
+/** \brief 128 bits block size */
+#define VA_PC_BLOCK_SIZE_128                0x00000001
+/** \brief 192 bits block size */
+#define VA_PC_BLOCK_SIZE_192                0x00000002
+/** \brief 256 bits block size */
+#define VA_PC_BLOCK_SIZE_256                0x00000004
+
+/** @name Attribute values for VAConfigAttribProtectedContentCipherMode */
+/** \brief AES CTR */
+#define VA_PC_CIPHER_MODE_CTR               0x00000001
+/** \brief AES CBC */
+#define VA_PC_CIPHER_MODE_CBC               0x00000002
```

Comment:
These are getting changed from the existing values below, can we leave them the same?

/** attribute values for VAConfigAttribContentProtectionCipherMode */
#define VA_PC_CIPHER_MODE_CBC 0x00000002
#define VA_PC_CIPHER_MODE_CTR 0x00000004

Reply:
Updated, thanks.